package.json

```diff
@@ -37,6 +37,9 @@
         "test:watch": "jest --watch",
         "test": "jest"
     },
+    "resolutions": {
```

Comment:
I think we need to remove this as well.

Reply:
good catch! i just removed the resolutions part